.github/workflows/ci.yaml

```diff
@@ -0,0 +1,50 @@
+name: ci
+
+on:
+  pull_request:
+    branches:
+      - '**'
+  push:
+    branches:
+      - master
+      - main
+      - 'release-*'
+
+jobs:
+  mri:
+    strategy:
+      matrix:
+        os: [ubuntu]
+        php:
+          - '5.3'
+          - '5.4'
+          - '5.5'
+          - '5.6'
+          - '7.0'
+          - '7.1'
+          - '7.2'
+          - '7.3'
+          - '7.4'
+          - '8.0'
+          - '8.1'
+    runs-on: ${{ matrix.os }}-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+        tools: composer:v2
```

Comment:
```suggestion
          tools: composer:v2
```

I tested this workflow in my public repo and it's pointed like this:
https://github.com/yuya-takeyama/github-actions-sandbox/actions/runs/1273102381/workflow
![image](https://user-images.githubusercontent.com/241542/134774556-ff2f3271-519b-4db3-b6dc-1905e9599233.png)